packages/e2e-tests/kit-node/__tests__/kit.spec.ts

```diff
@@ -307,13 +310,22 @@ describe('kit-node', () => {
 						'esm-env', // first added by svelte-kit
 						'esm-env', // second added by vite-plugin-svelte
 						'@sveltejs/kit',
-						'@sveltejs/adapter-node'
+						'@sveltejs/adapter-node',
+						'e2e-test-dep-svelte-nested-workspace-devdep',
+						'e2e-test-dep-svelte-simple'
 					],
 					`ssr.noExternal in ${filename}`
 				);
+				const expectedExcludes = ['@sveltejs/kit', '$app', '$env'];
+				if (!isServe) {
+					expectedExcludes.push(
```

Comment:
optimizeDeps.exclude during build isn't really needed anymore as newer vite no longer supports it, but a change for that would have to come in vitefu and that still supports older versions that have it.

i wonder if we should omit optimizeDeps config in the plugin instead of generating values that are ignored.